### DISCUSSION TITLE
Limit exceeded

### DISCUSSION THREAD

**Discussion by MSWallack** [TumblThreeApp/TumblThree#449] [Q&A (Question & Answer)]
For the last few weeks, I've been encountering "limit exceeded" errors (and usually, when I get the error, I can't download anything more until the next day). What are the ideal settings to avoid getting these errors?

**Comment by thomas694** [comment]
If you only get these errors recently, it could mean that you added more blogs, your intervals between blog updates are longer, the blogs publish more posts or the like. And it depends a bit whether you are downloading the blogs with the API or SVC crawler.
When you get these errors, you can try to reconnect your internet.

I'm using the default values for 'time' settings and changed the values for 'concurrent' ones to 4/2/2. For me it works well.

**Reply by Spezix** [reply]
Thank you for the post. I will try your settings.

**Reply by MSWallack** [reply]
What are the preferred settings beyond the 4/2/2 that you mention? And can you explain the difference between the API and SVC crawlers and why I should choose one or the other?

**Reply by thomas694** [reply]
Limiting both crawler types to 90 connections per 60 seconds.

The API and SVC crawlers are two ways how the app downloads things from Tumblr. With normal blogs you can switch to SVC, for the rest the app selects it automatically. E.g. a photography blog with large high-res photos may require switching to SVC, but most of the time you can stay with the preselected API. SVC crawler is needed for hidden blogs, Likes, LikedBy and Tag Searches and requires to be logged in.
